### DISCUSSION TITLE
Next new/empty workspace

### DISCUSSION THREAD

**Discussion by asoymos** [i3/i3#5962] [Q&A] [answered]
Hi and thank you, i3 is amazing. I have been struggling with this as I realized I didn't like to base my workflow on workspace numbers. Mostly I only need nested windows in groups, so with a keyboard shorcut to navigate between active workspaces (workspace next_on_output) and another to go to next empty workspace I can finally forget about numbers and just focus on my work. The thing is I have searched and haven't found an built in way of doing this, there is? (currently I solved it with [this script](https://github.com/i3/i3/issues/5484) but it would be nice to know if this kind of workflow is supported) Thank you.

**Comment by okraits** [comment, marked as answer]
I guess you mean the script of @orestisfl.

No, there is no command in i3 to do that. It's i3's philosophy to provide the IPC interface to implement such commands as scripts.

**Reply by asoymos** [reply]
Ok, thank you.
